tools/ds-identify

```diff
@@ -1565,6 +1565,25 @@ dscheck_VMware() {
     return "${DS_NOT_FOUND}"
 }
 
+dscheck_WSL() {
+    if [ ! "${DI_VIRT}" = "wsl" ]; then
```

Comment:
WSL only supports Linux, right? Since ds-identify runs on other kernels besides just the linux kernel, lets add an early check to based on information we already know to avoid the grep on non-Linux.

```suggestion
    if [ ! "${DI_VIRT}" = "wsl" -o "$DI_UNAME_KERNEL_NAME" != "Linux" ]; then
```

cc @igalic

Reply:
yeah, if we don't, everything else will also needlessly fail

Reply:
the l in wsl stands for Linux, so no other platform should match that, to begin with